tests/php/Controllers/Root/RootControllerTest.php

```diff
@@ -0,0 +1,206 @@
+        // 失敗時はエラーメッセージが表示される
+        $this->assertNull($getAssignData['tabbarParams']['docs'], 'error:リクエストがない場合にtabbarParamsのdocsがnullになっていない');
+    }
+
+    /**
+     * topActionでリクエストがない場合、
+     * アサイン情報のtabbarParamsの中のdevがnullになることを確認
+     * 
+     */
+    public function testTopActionIsNullDev() {
+        $rootController = new RootController;
+        $request = new Request();
+        $topAction = $rootController->topAction($request);
+        // Viewオブジェクトにアサインされている情報を取得する。
+        $getAssignData = $topAction->getData();
+        // 失敗時はエラーメッセージが表示される
+        $this->assertNull($getAssignData['tabbarParams']['dev'], 'error:リクエストがない場合にtabbarParamsのdevがnullになっていない');
+    }
+
+    /**
+     * topActionでhome=topというリクエストがあった場合、
+     * アサイン情報のtabbarParamsの中のhomeがtopになることを確認
+     * 
+     */
+    public function testTopActionIsHomeParamEqualTop() {
+        $rootController = new RootController;
+        $request = new Request(['home' => 'top']);
+        $topAction = $rootController->topAction($request);
+        // Viewオブジェクトにアサインされている情報を取得する。
+        $getAssignData = $topAction->getData();
+        // 失敗時はエラーメッセージが表示される
+        $this->assertEquals($getAssignData['tabbarParams']['home'], 'top', 'error:home=topのリクエストのとき場合にtabbarParamsのhomeがtopになっていない');
+    }
+
+        /**
+     * topActionでhome=hogefugaというリクエストがあった場合、
+     * アサイン情報のtabbarParamsの中のhomeがhogefugaになることを確認
+     * 
+     */
+    public function testTopActionIsHomeParamEqualHogefuga() {
+        $rootController = new RootController;
+        $request = new Request(['home' => 'hogefuga']);
+        $topAction = $rootController->topAction($request);
+        // Viewオブジェクトにアサインされている情報を取得する。
+        $getAssignData = $topAction->getData();
+        // 失敗時はエラーメッセージが表示される
+        $this->assertEquals($getAssignData['tabbarParams']['home'], 'hogefuga', 'error:home=topのリクエストのとき場合にtabbarParamsのhomeがtopになっていない');
+    }
+
+    /**
+     * topActionで
+     * アサイン情報のアサイン情報にtabbarStatusが存在することを確認
+     * 
+     */
+    public function testTopActionIsKeyTabbarStatus() {
+        $rootController = new RootController;
+        $request = new Request();
+        $topAction = $rootController->topAction($request);
+        // Viewオブジェクトにアサインされている情報を取得する。
+        $getAssignData = $topAction->getData();
+        // 失敗時はエラーメッセージが表示される
+        $this->assertArrayHasKey('tabbarStatus', $getAssignData, 'error:アサイン情報としてtabbarStatusが存在しない');
+    }
+
+    /**
+     * topActionで
+     * アサイン情報のtabbarStatusがhomeであることを確認する
+     * 
+     */
+    public function testTopAction() {
+        $rootController = new RootController;
+        $request = new Request();
+        $topAction = $rootController->topAction($request);
+        // Viewオブジェクトにアサインされている情報を取得する。
+        $getAssignData = $topAction->getData();
+        // 失敗時はエラーメッセージが表示される
+        $this->assertEquals($getAssignData['tabbarStatus'], 'home', 'error:アサイン情報としてtabbarStatusはhomeではない');
+    }
+
+}
```

Comment:
__[code_sniffer]__
* The closing brace for the class must go on the next line after the body